core/src/main/java/dev/hypera/scaffolding/instance/SchematicChunkLoader.java

```diff
@@ -114,7 +94,8 @@ public static class Builder {
         private int yOffset;
         private int zOffset;
 
-        private Builder() {}
+        private Builder() {
+        }
```

Comment:
There should be an empty line above this line.

Reply:
there is an empty line?